apps/platform/src/components/Facets/facets.types.ts

```diff
@@ -0,0 +1,67 @@
+export enum ENTITY {
+  TARGET = "target",
+  DISEASE = "disease",
+}
+
+export const ALL_CATEGORY = "All Categories";
+
+export const TARGET_CATEGORIES = {
+  [ALL_CATEGORY]: "All",
+  Names: "Approved Name",
+  Symbol: "Approved Symbol",
+  "ChEMBL Target Class": "ChEMBL Target Class",
+  "GO:BP": "GO:BP",
+  "GO:CC": "GO:CC",
+  "GO:MF": "GO:MF",
+  Reactome: "Reactome",
+  "Subcellular Location": "Subcellular Location",
+  "Target ID (ENSG)": "Target ID",
+  "Tractability Antibody": "Tractability Antibody",
+  "Tractability Other Modalities": "Tractability Other Modalities",
+  "Tractability PROTAC": "Tractability PROTAC",
+  "Tractability Small Molecule": "Tractability Small Molecule",
+};
+
+export const DISEASE_CATEGORIES = {
+  [ALL_CATEGORY]: "All",
+  Disease: "Disease",
+  "Therapeutic Area": "Therapeutic Area",
+};
```

Comment:
Consistency here

Reply:
this is automatically formatted by prettier, as disease is a dingle word it does not need string literals and `ALL_CATEGORY` is a variable as it is being used in different places